src/toolkit/facility_cost.cycpp.h

```diff
@@ -94,18 +94,25 @@ std::unordered_map<std::string, double> GenerateParamList() const override {
 
 /// @brief Calculates the levelized unit cost of production, accounting for
 /// capital depreciation, O&M, labor, property taxes, and input costs.
-/// 
-/// unit_cost = cost_override if cost_override > 0, otherwise unit_cost = production_cost + material_cost
-/// 
+///
+/// unit_cost = cost_override if cost_override > 0, otherwise unit_cost =
+/// production_cost + material_cost
```

Comment:
Will clang override it if we add some indenting in here?
```suggestion
///             production_cost + material_cost
```

Reply:
And maybe a few other places